deepmd/pt/infer/deep_eval.py

```diff
@@ -130,7 +130,8 @@ def __init__(
                         ] = state_dict[item].clone()
                 state_dict = state_dict_head
             model = get_model(self.input_param).to(DEVICE)
-            model = torch.jit.script(model)
+            if "Hessian" not in str(type(model)):
+                model = torch.jit.script(model)
```

Comment:
Please try [`torch.jit.ignore`](https://pytorch.org/docs/stable/generated/torch.jit.ignore.html) and see if it works rather than making a special usage here.

Reply:
I tried using `torch.jit.ignore`, but it doesn't work as expected. The method `_cal_hessian_all` cannot be compiled by `torch.jit.script`. While applying `@torch.jit.ignore` to `_cal_hessian_all` allows `EnergyHessianModel` to pass the JIT script compilation, the issue arises when performing the forward pass. Specifically, when calculating the batch output, the `RecursiveScriptModule` throws a `RuntimeError` because it cannot simultaneously compute JIT-compiled energy and force outputs along with the JIT-ignored Hessian.

As a result, I did not apply `@torch.jit.ignore` to `_cal_hessian_all`. Instead, I opted for a `try-except` block when calling `torch.jit.script(model)`, which serves as a workaround for this issue and allows for the special usage scenario.